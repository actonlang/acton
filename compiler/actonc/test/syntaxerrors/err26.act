# Multiple statements on same line without semicolon
def multiple():
    x = 1 y = 2