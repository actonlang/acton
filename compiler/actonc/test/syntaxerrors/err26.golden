Building file test/syntaxerrors/err26.act using temporary scratch directory
[error Parse error]: unexpected "y "
                     expecting ';', call arguments, slice/index expression, comma, end of line, if clause, or operator

     +--> test/syntaxerrors/err26.act@3:11-3:12
     |
   3 |     x = 1 y = 2
     :           ^ 
     :           `- unexpected "y "
     :              expecting ';', call arguments, slice/index expression, comma, end of line, if clause, or operator
     :              
-----+
